modin/pandas/test/test_dataframe.py

```diff
@@ -5181,8 +5181,10 @@ def test_iterrows(self, data):
             df_equals(pandas_series, modin_series)
             assert pandas_index == modin_index
 
+    @pytest.mark.parametrize("name", [None, "NotPandas", "Pandas"])
+    @pytest.mark.parametrize("index", [True, False])
     @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
-    def test_itertuples(self, data):
+    def test_itertuples(self, name, index, data):
```

Comment:
could you also add a test case where `df.columns` is `MultiIndex`? It seems that your implementation would behave differently from pandas in this case

Reply:
Added test